Packs/CommonScripts/Scripts/IsIPPrivate/IsIPPrivate.py

```diff
@@ -0,0 +1,116 @@
+from ipaddress import IPv4Address, IPv4Network
+from time import sleep
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+DELIMETER = ","
+POLLING_TIME = 2  # Time in seconds to wait for indicator indexing
+
+
+def check_ip_internal(ip: str, ranges: list[str]):
+    try:
+        return any((IPv4Address(ip) in IPv4Network(cidr.split(DELIMETER)[0] if DELIMETER in cidr else cidr) for cidr in ranges))
+    except ValueError:
+        demisto.log("One or more IP ranges or IPs are invalid. Please make sure the list is in the correct structure.")
+        return True
+
+
+def get_ip_tag(ip: str, ranges: list[str]):
+    for cidr in ranges:
+        tag = None
+        if DELIMETER in cidr:
+            cidr, tag = cidr.split(DELIMETER)[0], cidr.split(DELIMETER)[1].strip()
+        try:
+            if IPv4Address(ip) in IPv4Network(cidr):
+                return tag
+        except ValueError:
+            if ip == cidr:
+                return tag
+    return None
+
+
+def create_markdown_table(ip_list):
+    ip_list = sorted(ip_list, key=lambda x: not x['Private'])
+    markdown_string = "| Address | Private | Tag |\n| --- | --- | --- |\n"
+    for ip in ip_list:
+        markdown_string += f"| {ip['Address']} | {ip['Private']} | {ip['Tag']}|\n"
+    return markdown_string
+
+
+ranges_list_name = demisto.args().get("PrivateIPsListName", "PrivateIPs")
+ip_addresses_to_check = argToList(demisto.args().get("IPAddresses", None))
```

Comment:
```suggestion
args = demisto.args()
ranges_list_name = args.get("PrivateIPsListName", "PrivateIPs")
ip_addresses_to_check = argToList(args.get("IPAddresses"))
```
get default value is None